odoo/addons/base/models/res_users.py

```diff
@@ -803,6 +803,11 @@ def _is_public(self):
         self.ensure_one()
         return self.has_group('base.group_public')
 
+    @api.multi
+    def _is_portal(self):
```

Comment:
Side note: use share field of user model, it is its purpose to be "not employees" (which is better than checking portal group if people have multi portal groups).

Reply:
Thank you, I wasn't aware of this.